text.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] Kindly make sure this file `test.md` is not in your repository because the project requirements require that this file should be ignored by git so it means you must make sure that before you push your code from your local computer, it must be presented into `gitignore` file and make sure that it is not represented in your repository anymore. Kindly fix that.